exercises/flatten-array/src/test/java/FlattenerTest.java

```diff
@@ -32,6 +32,7 @@ public void testASingleLevelOfNestingWithNoNulls() {
               8)));
     }
 
+    @Ignore
```

Comment:
As @stkent noted, going to need an `import org.junit.Ignore;` at the top to prevent a compilation error for practitioners when they download the exercise.